python/ray/data/read_api.py

```diff
@@ -1036,6 +1036,20 @@ def read_json(
         >>> ds.take(1)
         [{'order_number': 10107, 'quantity': 30, 'year': '2022', 'month': '09'}]
 
+        When reading large files, the default block size configured in PyArrow can be too small,
+        resulting in the following error:
+        ``pyarrow.lib.ArrowInvalid: straddling object straddles two block boundaries
+        (try to increase block size?)``.
+
+        To resolve this, use the ``read_options`` parameter to set a larger block size:
+
+        >>> import pyarrow.json as pajson
+        >>> block_size = 10 << 20 # Set block size to 10MB
+        >>> ray.data.read_json(
+        ...     s3://anonymous@ray-example-data/log.json,
```

Comment:
missing the `""` - "s3://anonymous@ray-example-data/log.json"

Reply:
thanks for the catch, just fixed